Ynab/Sanitisers/MilliunitSanitiser.cs

```diff
@@ -19,6 +19,7 @@ public static class MilliunitSanitiser
         return null;
     }
     
+    // TODO: This is a poor naming convention.
     public static int Desanitise(decimal currency)
```

Comment:
Rename this to 'ConvertTo' and 'ConvertFrom' maybe. Might even be able to hook into the 'Converter...' APIs in .NET if they're exposed.